server/bot/imports/types.ts

```diff
@@ -64,3 +68,180 @@ export type CommandOptions = {
     roleIDs?: string[]
   }
 }
+
+export class TriviaSession {
```

Comment:
I feel like this whole class should go under `server/bot/commands/trivia.ts` file and not here.